src/pages/security-contact-vulnerability-disclosure.php

```diff
@@ -26,4 +26,9 @@
 <strong>Backdoor Insertion Proof-of-Concept Bounty:</strong> The first time
 someone tricks me into inserting the string "BackdoorPoCTwitter" into a release
 of any of my software projects, I will pay them $100 USD.
-</p>
+
+<p>
+First winner: @Sc00bzT (https://twitter.com/Sc00bzT/status/730903007014076416)
+<br/><br/>
+Offer still stands!
+</p>
(BackdoorPoCTwitter)
```

Comment:
ah looks like github fixed their bug. this was hidden from the GH diff when I submitted it 👍